src/Authentication/ManagedCredentialsAuthenticator.php

```diff
@@ -39,13 +39,13 @@ public function getAuthenticationToken(): string
                 sprintf(
                     '/metadata/identity/oauth2/token?api-version=%s&format=text&resource=%s',
                     self::API_VERSION,
-                    $this->resource
+                    $this->resource,
```

Comment:
nejdřív si ten rule dá do ignoru a pak tam ty čárky stejně dopíše :) nebo obráceně?

Reply:
tak to asi můžeš z toho ignore vyhodit

Reply:
Dopsal jsem čárky ve **volání** funkcí (to je podporované od 7.3), ale ignore je pro trailing commas v **deklaraci** funkcí, to je až od PHP 8.0.

No každopádně když teď můžu 7.x vyhodit, tak to z ignore taky vyhodím a dodělám trailing commas i v deklaracích. 🙂

Reply:
ahtak